src/uct/ib/base/ib_iface.c

```diff
@@ -577,21 +592,20 @@ int uct_ib_iface_is_reachable(const uct_iface_h tl_iface,
     uct_ib_iface_t *iface           = ucs_derived_of(tl_iface, uct_ib_iface_t);
     int is_local_eth                = uct_ib_iface_is_roce(iface);
     const uct_ib_address_t *ib_addr = (const void*)dev_addr;
-    uct_ib_unpack_address_t unpack_addr;
+    uct_ib_address_pack_params_t params;
 
-    uct_ib_address_unpack(ib_addr, &unpack_addr);
+    uct_ib_address_unpack(ib_addr, &params);
 
     if (/* at least one PKEY has to be with full membership */
-        (((unpack_addr.pkey | iface->pkey) & UCT_IB_PKEY_MEMBERSHIP_MASK) == 0) ||
+        !(((params.pkey | iface->pkey) & UCT_IB_PKEY_MEMBERSHIP_MASK)) ||
```

Comment:
seems like one pair of ( ) can be removed

Reply:
yes, removed